gcp/app-engine/main.py

```diff
@@ -74,6 +86,15 @@ def getconn():
 
 @app.route(PUBLIC_API_PREFIX + '/tournaments', methods=['GET'])
 def get_tournaments():
+    decoded_token = None
```

Comment:
add auth to endpoints -- maybe in another MR we should add this as an decorator or something to automatically apply auth to endpoints

Reply:
NOTE: this has been added in a follow-up MR